crawler/file_processing.py

```diff
@@ -272,7 +272,7 @@ def __init__(self, file_name: str, centre: Centre):
 
         # These are to allow some variability in headers,
         # due to receiving inconsistent file formats
-        self.header_regex_correction_dict = {"Root Sample": FIELD_ROOT_SAMPLE_ID}
+        self.header_regex_correction_dict = {r"^\s*Root Sample\s*$": FIELD_ROOT_SAMPLE_ID}
```

Comment:
I think we should strip the fieldnames first before looking for matches to make our regex simpler. This would require a little rework in the crawler though so you could just do it for this change to start?

Reply:
Yes that would be nicer, will take a look at what I can change here